int32 command_lifetime
---
